src/test/client/ops.cc

```diff
@@ -0,0 +1,43 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+/*
+ * Ceph - scalable distributed file system
+ *
+ * Copyright (C) 2022 Red Hat
+ *
+ * This is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License version 2.1, as published by the Free Software
+ * Foundation.  See file COPYING.
+ *
+ */
+
+#include <iostream>
+#include <errno.h>
+#include "TestClient.h"
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+#include "gtest/gtest-spi.h"
+#include "gmock/gmock-matchers.h"
+#include "gmock/gmock-more-matchers.h"
+
+using ::testing::AnyOf;
+using ::testing::Gt;
+using ::testing::Eq;
+
+TEST_F(TestClient, CheckDummyOP) {
+  ASSERT_EQ(client->check_dummy_op(myperm), -EOPNOTSUPP);
+}
+
+TEST_F(TestClient, CheckUnknownSessionOp) {
+  ASSERT_EQ(client->send_unknown_session_op(-1), 0);
+  sleep(5);
```

Comment:
no need to sleep and wait here.

Reply:
I think patrick once commented to have atleast some sleep here. Im unsure of the reason though

Reply:
https://github.com/ceph/ceph/pull/47080#discussion_r948112959

Reply:
Oh, I've used `wait_for_latest_osdmap`, we don't need that sleep anymore maybe?

Reply:
I checked it, we need to wait before requesting new map else the test gets stuck.

Reply:
Won't the `objecter->wait_for_latest_osdmap(ca::use_blocked[ec]);` wait it ?

Reply:
Ran tests without sleep and it got stuck, that asserts that It is not waiting for it.

Reply:
Okay, that won't wait for a osdmap with a new epoch, may still get the osdmap with old epoch.